docs/src/templates/modal.js

```diff
@@ -460,7 +466,7 @@ export default class ModulesModal extends React.Component {
                 name: 'title',
                 type: 'string',
                 default: '',
-                description: 'Optinal string to give a Modal a title. If it is an empty then Modal\'s header (with a close button) will not be shown.',
+                description: 'Optinal string to give a Modal a title. If it is an empty (and closeButton is not defined) then Modal\'s header will not be shown.',
```

Comment:
"Optional" is misspelled here.  I do realize this was an existing issue, but since we're updating it, it be nice to fix it if you don't mind.

Reply:
done

Reply:
Thanks @raoufr